src/components/SenderDropdownItem.tsx

```diff
@@ -0,0 +1,112 @@
+// Copyright 2021 Parity Technologies (UK) Ltd.
+// This file is part of Parity Bridges UI.
+//
+// Parity Bridges UI is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Parity Bridges UI is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Parity Bridges UI.  If not, see <http://www.gnu.org/licenses/>.
+
+import React, { useState } from 'react';
+import cx from 'classnames';
+import { Box, makeStyles, Typography } from '@material-ui/core';
+import shorterItem from '../util/shortenItem';
+import AccountIdenticon from './AccountIdenticon';
+import { useGUIContext } from '../contexts/GUIContextProvider';
+
+interface Props {
+  name: string;
+  balance: string;
+  companionBalance: string;
+  address: string;
+  showCompanion: boolean;
+}
+
+const useStyles = makeStyles((theme) => ({
```

Comment:
@goldsteinsveta i wonder if there is a better way to define styles and `Box` component hierarchy